docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -0,0 +1,248 @@
+
+This model will allow us to understand and quantify the impact of different combinations of preventive and treatment strategies on wasting and wasting attributable DALYs at the national level. 
+
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+
+The primary objective of this project is to answer the following question: what is the cost and impact of combinations of preventive and therapeutic strategies for reducing overall wasting prevalence and eliminating SAM in Ethiopia? 
+We will use data from the 2019 GBD and published literature to inform the parameters for our simulation. We will simulate the changes in MAM and SAM disease incidence, prevalence and mortality from 2022 to 2027 in response to a combination of preventive and therapeutic treatment interventions in Ethiopia.
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+ .. note::
+    link to DAGs page
+    use round circles with DAGs
+
+**Outcome (O)**:
+
+
+
+**Most proximal determinant/exposure (E)**:
+
+
+
+**Confounders (C)**:
+
+
+
+**Effect modifiers**:
+
+
+**Mediators (M)**:
+
+
+.. _3.2:
+
+3.2 Effect sizes
+----------------
+
+
+
+4.0 Intervention
+++++++++++++++++
+
+Historically, prevention research has primarily focused on stunting, and, as a research outcome, wasting has been considered primarily within the context of humanitarian emergencies. Although the volume of studies related to wasting prevention through direct and indirect health-care sector areas has increased in recent years, this evidence base is mixed and often inconclusive8. We reviewed the literature from the recent Keats et al 2021 update of effective interventions to address maternal and child malnutrition and selected interventions that have moderate or strong evidence for implementation5. We selected interventions that:  
+
+1)	Directly prevent acute malnutrition, moderate or severe;
+2)	Treat or manage acute malnutrition, moderate or severe;
+3)	Increase rates of exclusive or continued breastfeeding;
+4)	Increase birthweight;
+5)	Reduce incidence of infectious disease; or
+6)	Improve recovery from infectious disease.
+
+
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+
+**Baseline**
+The baseline scenario will project GBD 2019 demographic and disease trends out from 2022 to 2027 and coverage rates for all preventive and therapeutic interventions will be held constant across the 5 years of the microsimulation to simulate a business-as-usual treatment scenario. 
+
+**Alternative**
```

Comment:
Just a thought - should we have 2 alternative scenario sub-headings here to make it clear we will be running 3 scenarios?

Reply:
Good idea: **Alternative Scenario 1** and **Alternative Scenario 2** will be clearer to the busy reader.